spec/factories/teams.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :team do
     sequence(:name) { |n| "Patriots#{n}" }
-    city "New England"
+    location "New England"
```

Comment:
This was the reason for that other lone test error after the test db migration error from the other day. I thought I caught all mentions of `city` but missed this one. I forgot that factories do not use symbols so it didn't stick out to me.  

Should I have added this to the `create_teams` branch?
- Thinking about it now, that probably would have been the more logical thing to do since this change has nothing to do with weeks or seasons.